packages/auth/package.json

```diff
@@ -16,7 +16,7 @@
     "@auth/core": "^0.16.1",
     "@auth/prisma-adapter": "^1.0.4",
     "@t3-oss/env-nextjs": "^0.7.0",
-    "next": "^13.5.4",
+    "next": "^14.2.3",
     "next-auth": "0.0.0-manual.b53ca00b",
```

Comment:
What is this versioning? Did it come like this 0.o 

Never seen anytthing like this.

Reply:
It came with 13.5 but I just updated it to the latest version